deployment/cdk/opensearch-service-migration/options.md

```diff
@@ -1,52 +1,89 @@
+| ---------- | ------- | ----------------------- | :---------------------------------------------------------------------------------------------------------------------------------------- |
+| vpcEnabled | boolean | true                    | Enable VPC to place Domain and Migration resources in. If a `vpcId` is not provided a new VPC will be created                             |
+| vpcId      | string  | "vpc-123456789abcdefgh" | Specify an existing VPC to place the domain inside of                                                                                     |
+| vpcAZCount | number  | 2                       | The number of Availability Zones for the created VPC. **Note**: Only applicable if creating a new VPC (thus `vpcId` must not be provided) |
+
+### MSK(Kafka) Options
+
+| Name                             | Type         | Example                                                                                                 | Description                                                                                                                                                                                        |
+| -------------------------------- | ------------ | ------------------------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| mskBrokersPerAZCount             | number       | 1                                                                                                       | The number of broker nodes per MSK Availability Zone                                                                                                                                               |
+| mskSubnetIds                     | string array | ["subnet-123456789abcdefgh", "subnet-223456789abcdefgh"]                                                | Specify the subnet IDs of an existing VPC to place MSK brokers in. **NOTE** MSK currently requires either 2 or 3 subnets to be specified and EACH subnet should use a different Availability Zone. |
+| mskAZCount                       | number       | 2                                                                                                       | The number of Availability Zones for the MSK cluster to use. **NOTE** This value must be 2 or 3                                                                                                    |
+| mskARN (Not currently available) | string       | `"arn:aws:kafka:us-east-2:12345678912:cluster/msk-cluster-test/81fbae45-5d25-44bb-aff0-108e71cc079b-7"` | Supply an existing MSK cluster ARN to use. **NOTE** As MSK is using an L1 construct this is not currently available for use                                                                        |
+
+## Options being deprecated
```

Comment:
"The term deprecated refers to a functionality that still exists in software whose use is not recommended"

[Source](https://www.baeldung.com/cs/deprecated-vs-obsolete)

With this definition I'd say these options are depreciated and will be removed in a future release.